src/IMU.cpp

```diff
@@ -1,11 +1,56 @@
 #include <IMU.hpp>
 
+namespace IO = EVT::core::IO;
+
 namespace IMU {
+IMU::IMU(IO::I2C& i2c) : bno055(i2c) {
+    bno055.setup();
+}
+
 CO_OBJ_T* IMU::getObjectDictionary() {
     return objectDictionary;
 }
 
 uint16_t IMU::getObjectDictionarySize() const {
     return OBJECT_DICTIONARY_SIZE;
 }
+
+void IMU::cycle() {
+    // Retrieve the Euler X, Y and Z values from the bno055
+    uint16_t eulerX, eulerY, eulerZ;
+
+    bno055.getEuler(eulerX, eulerY, eulerZ);
```

Comment:
It's a little out of scope for this task, but while you're at it, you could just pass the CAN values like `vectorY[0]` in right here to get it writing to CAN. That's all we really need to get IMU main ready to go